src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
```

Comment:
It's a good practice to create variables for reusable data, but let's create other variables for discounts (50 and 20) an use them instead of 'magic numbers'